WordPress/Classes/ViewRelated/System/WPTabBarController+Swift.swift

```diff
@@ -0,0 +1,104 @@
+
+// MARK: - Tab Access Tracking
+
+extension WPTabBarController {
+    private static let tabIndexToStatMap: [WPTabType: WPAnalyticsStat] = [
+        .mySites: .mySitesTabAccessed,
+        .reader: .readerAccessed,
+        .me: .meTabAccessed
```

Comment:
Do you know offhand why `notifications_accessed` isn't part of this?

Reply:
Not sure, it's in `viewWillAppear` of `NotificationsViewController`. I wonder we should do it this way instead of putting these in `WPTabBarController`.

Reply:
I think the tab bar is the right solution .. we're firing off `notifications_accessed` every time the view is presented including after tapping a notification to see detail and then tapping back. I bet we're over-counting notifications because of this.

Reply:
Ah, yeah, you're right. I added that in #11446.